### PR TITLE
Add Packer build template for WordPress

### PR DESCRIPTION
**Pull request by countingtoten**
Add the Packer build templates for wordpress-18-04 and wordpress-20-04

### CODE REVIEW

File: wordpress-18-04/scripts/010-php.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+update-alternatives --set php /usr/bin/php7.4
+
+sed -e "s|upload_max_filesize.*|upload_max_filesize = 16M|g" \
+    -e "s|post_max_size.*|post_max_size = 16M|g" \
+    -e "s|max_execution_time.*|max_execution_time = 60|g" \
+    -i /etc/php/7.4/apache2/php.ini
```
Comment (keladhruv) [inline wordpress-18-04/scripts/010-php.sh:8]: Extra line

File: wordpress-18-04/scripts/011-wordpress.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+tar -C /var/www \
+    -xvvf /tmp/wordpress.tar.gz
+
+wpfail2ban="wp-fail2ban.${fail2ban_version}.zip"
+wget https://downloads.wordpress.org/plugin/${wpfail2ban} -O /tmp/wp-fail2ban.zip
+unzip /tmp/wp-fail2ban.zip -d /tmp/
+
+# install the fail2ban bits (plugin itself installed in first-login script)
+mkdir -p /etc/fail2ban/filter.d
+cp -auv  /tmp/wp-fail2ban/filters.d/* /etc/fail2ban/filter.d
```
Comment (keladhruv) [inline wordpress-18-04/scripts/011-wordpress.sh:18]: Same

File: wordpress-18-04/scripts/012-apache.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+chown -R www-data: /var/log/apache2
+chown -R www-data: /etc/apache2
+chown -R www-data: /var/www
+
+# Enable re-write
+(cd /etc/apache2/mods-enabled/ &&
+ ln -vs ../mods-available/rewrite.load . )
```
Comment (keladhruv) [inline wordpress-18-04/scripts/012-apache.sh:9]: same

File: wordpress-20-04/scripts/012-apache.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+chown -R www-data: /var/log/apache2
+chown -R www-data: /etc/apache2
+chown -R www-data: /var/www
+
+# Enable re-write
+(cd /etc/apache2/mods-enabled/ &&
+ ln -vs ../mods-available/rewrite.load . )
```
Comment (keladhruv) [inline wordpress-20-04/scripts/012-apache.sh:9]: same

File: wordpress-20-04/scripts/011-wordpress.sh [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+tar -C /var/www \
+    -xvvf /tmp/wordpress.tar.gz
+
+wpfail2ban="wp-fail2ban.${fail2ban_version}.zip"
+wget https://downloads.wordpress.org/plugin/${wpfail2ban} -O /tmp/wp-fail2ban.zip
+unzip /tmp/wp-fail2ban.zip -d /tmp/
+
+# install the fail2ban bits (plugin itself installed in first-login script)
+mkdir -p /etc/fail2ban/filter.d
+cp -auv  /tmp/wp-fail2ban/filters.d/* /etc/fail2ban/filter.d
```
Comment (keladhruv) [inline wordpress-20-04/scripts/011-wordpress.sh:18]: same

File: wordpress-20-04/scripts/010-php.sh [hunk 6]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+sed -e "s|upload_max_filesize.*|upload_max_filesize = 16M|g" \
+    -e "s|post_max_size.*|post_max_size = 16M|g" \
+    -e "s|max_execution_time.*|max_execution_time = 60|g" \
+    -i /etc/php/7.4/apache2/php.ini
```
Comment (keladhruv) [inline wordpress-20-04/scripts/010-php.sh:6]: same

**Review by keladhruv — CHANGES_REQUESTED** [review]

File: wordpress-18-04/template.json [hunk 7]
```diff
@@ -0,0 +1,88 @@
+
+{
+  "variables": {
+    "do_api_token": "{{env `DIGITALOCEAN_API_TOKEN`}}",
+    "image_name": "wordpress-18-04-snapshot-{{timestamp}}",
+    "apt_packages": "apache2 fail2ban libapache2-mod-php mysql-server php7.4 php7.4-apcu php7.4-bz2 php7.4-curl php7.4-gd php7.4-gmp php7.4-intl php7.4-json php7.4-mbstring php7.4-mysql php7.4-pspell php7.4-soap php7.4-tidy php7.4-xml php7.4-xmlrpc php7.4-xsl php7.4-zip postfix python-certbot-apache software-properties-common unzip",
+    "appiication_name": "WordPress",
+    "appiication_version": "5.5.1",
```
Comment (countingtoten) [inline wordpress-18-04/template.json:8]: We are going to explicitly select which version of WordPress we want to install. We should automate triggering a new build anytime a new WordPress version is released.

File: wordpress-20-04/template.json [hunk 8]
```diff
@@ -0,0 +1,86 @@
+
+{
+  "variables": {
+    "do_api_token": "{{env `DIGITALOCEAN_API_TOKEN`}}",
+    "image_name": "wordpress-20-04-snapshot-{{timestamp}}",
+    "apt_packages": "apache2 fail2ban libapache2-mod-php mysql-server php php-apcu php-bz2 php-curl php-gd php-gmp php-intl php-json php-mbstring php-mysql php-pspell php-soap php-tidy php-xml php-xmlrpc php-xsl php-zip postfix python3-certbot-apache software-properties-common unzip",
+    "appiication_name": "WordPress",
+    "appiication_version": "5.5.1",
```
Comment (countingtoten) [inline wordpress-20-04/template.json:8]: We are going to explicitly select which version of WordPress we want to install. We should automate triggering a new build anytime a new WordPress version is released.

**Pull request merged**